MantidPlot/src/Mantid/MantidUI.cpp

```diff
@@ -2043,39 +2043,38 @@ InstrumentWindow* MantidUI::getInstrumentView(const QString & wsName, int tab)
 
   //Need a new window
   const QString windowName(QString("InstrumentWindow:") + wsName);
-  InstrumentWindow *insWin = new InstrumentWindow(wsName,QString("Instrument"),appWindow(),windowName);
+  
   try
   {
-    insWin->init();
+    InstrumentWindow *insWin = new InstrumentWindow(
+        wsName, QString("Instrument"), appWindow(), windowName);
+
+    insWin->getInstrumentWidget()->selectTab(tab);
```

Comment:
Please fix this so that the client does not have to know about the `InstrumentWidget` i.e. `instWindow->selectTab(tab)`